### PR TITLE
Changed python tutorial to use longitude and latitude coordinates of ISS

### PR DESCRIPTION
**Pull request by autophagy**
@mechanomi Would you be able to take care of the screenshots? I can't replicate the nice drop shadow etc on my Linux machine :cry:

### CODE REVIEW

**Comment by nomicode** [comment]
thank you for this!!

let's wait until we've merged in https://github.com/crate/crate-tutorials/pull/5, so that we can sync up the wording

**Review by nomicode — APPROVED** [review]
thank you for this!!

my changes:

- rebased against `master`
- copied over wording/copy changes from node tutorial
- fixed node tutorial for consistency
- added basic exception handling to the python script
- fixed use of quotes to match `black` output

we discussed maybe updating this tutorial to use connection pooling

I'm happy for you to merge this as is, and then doing another pr to add connection pooling. but maybe you wanna add it to this one? up to you

**Comment by autophagy** [comment]
Ill do it in another one :)

**Pull request merged**
